### PR TITLE
Offramp

### PR DESCRIPTION
**Pull request by mateuszjablonski**

### CODE REVIEW

File: Sources/Ramp/Configuration.swift [hunk 1]
```diff
@@ -26,25 +36,27 @@ extension Configuration {
...
-        let url = url ?? Constants.defaultUrl
         guard var urlComponents = URLComponents(string: url) else { throw Error.invalidUrl }
         
-        urlComponents.appendQueryItem(name: "swapAsset", value: swapAsset)
-        urlComponents.appendQueryItem(name: "swapAmount", value: swapAmount)
+        urlComponents.appendQueryItem(name: "containerNode", value: containerNode)
+        urlComponents.appendQueryItem(name: "deepLinkScheme", value: deepLinkScheme)
+        urlComponents.appendQueryItem(name: "defaultAsset", value: defaultAsset)
+        urlComponents.appendQueryItem(name: "defaultFlow", value: defaultFlow?.rawValue)
+        urlComponents.appendQueryItem(name: "enabledFlow", value: enabledFlow?.map(\.rawValue).joined(separator: ","))
```
Comment (jakubsta) [inline Sources/Ramp/Configuration.swift:45]:
```suggestion
        urlComponents.appendQueryItem(name: "enabledFlows", value: enabledFlows?.map(\.rawValue).joined(separator: ","))
```

File: Sources/Ramp/Configuration.swift [hunk 2]
```diff
@@ -2,22 +2,32 @@ import Foundation
...
+    
+    /// main URL
+    public var url: String = Constants.defaultUrl
+    
+    /// query params
+    public var containerNode: String? = nil
+    public var deepLinkScheme: String? = nil
+    public var defaultAsset: String? = nil
+    public var defaultFlow: Flow? = nil
+    public var enabledFlow: [Flow]? = nil
```
Comment (jakubsta) [inline Sources/Ramp/Configuration.swift:15]:
```suggestion
    public var enabledFlows: [Flow]? = nil
```

**Review by jakubsta — CHANGES_REQUESTED** [review]

File: Sources/Ramp/Configuration.swift (hunk 1 above)
Comment (jakubsta) [reply Sources/Ramp/Configuration.swift:45]: please transform it to uppercase.

File: Ramp.podspec [hunk 3]
```diff
@@ -1,12 +1,13 @@
 Pod::Spec.new do |spec|
     spec.name = 'Ramp'
-    spec.version = '2.0.0'
+    spec.version = '3.0.0'
     spec.license = 'proprietary'
     spec.summary = 'Ramp SDK for iOS'
     spec.homepage = 'https://ramp.network/'
     spec.authors = { 'Ramp Network' => 'dev@ramp.network' }
-    spec.source = { :git => 'https://github.com/RampNetwork/ramp-sdk-ios', :tag => spec.version }
+    spec.source = { :git => 'https://github.com/RampNetwork/ramp-sdk-ios.git', :branch => 'offramp' }
```
Comment (mateuszjablonski) [inline Ramp.podspec:8]: don't forget to change it back when releasing

File: Sources/Ramp/Configuration.swift [hunk 4]
```diff
@@ -1,52 +1,82 @@
...
+    public var fiatCurrency: String?
+    public var fiatValue: String?
+    public var finalUrl: String?
+    public var hostApiKey: String?
+    public var hostAppName: String?
+    public var hostLogoUrl: String?
+    public var offrampWebhookV3Url: String?
+    public var selectedCountryCode: String?
+    public var swapAmount: String?
+    public var swapAsset: String?
```
Comment (jakubsta) [inline Sources/Ramp/Configuration.swift:24]:
```suggestion
    public var swapAsset: String?
    public var offrampAsset: String?
```

File: Sources/Ramp/Configuration.swift [hunk 5]
```diff
@@ -1,52 +1,82 @@
...
+        urlComponents.appendQueryItem(name: "finalUrl", value: finalUrl)
+        urlComponents.appendQueryItem(name: "hostApiKey", value: hostApiKey)
         urlComponents.appendQueryItem(name: "hostAppName", value: hostAppName)
-        urlComponents.appendQueryItem(name: "userEmailAddress", value: userEmailAddress)
+        urlComponents.appendQueryItem(name: "hostLogoUrl", value: hostLogoUrl)
+        urlComponents.appendQueryItem(name: "offrampWebhookV3Url", value: offrampWebhookV3Url)
         urlComponents.appendQueryItem(name: "selectedCountryCode", value: selectedCountryCode)
-        urlComponents.appendQueryItem(name: "defaultAsset", value: defaultAsset)
+        urlComponents.appendQueryItem(name: "swapAmount", value: swapAmount)
+        urlComponents.appendQueryItem(name: "swapAsset", value: swapAsset)
```
Comment (jakubsta) [inline Sources/Ramp/Configuration.swift:58]:
```suggestion
        urlComponents.appendQueryItem(name: "swapAsset", value: swapAsset)
        urlComponents.appendQueryItem(name: "offrampAsset", value: offrampAsset)      
```

**Pull request merged**